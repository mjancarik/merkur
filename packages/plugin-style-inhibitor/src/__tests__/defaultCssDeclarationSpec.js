import defaultCssDeclaration from '../defaultCssDeclaration';

describe('defaultCssDeclaration', () => {
  it('should match snapshot', () => {
    expect(defaultCssDeclaration).toMatchSnapshot();
  });
});
