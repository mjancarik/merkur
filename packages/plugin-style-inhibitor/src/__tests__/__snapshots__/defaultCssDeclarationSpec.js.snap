// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaultCssDeclaration should match snapshot 1`] = `
Object {
  "align-content": "normal",
  "align-items": "normal",
  "align-self": "auto",
  "alignment-baseline": "auto",
  "animation": "none",
  "animation-delay": "0s",
  "animation-direction": "normal",
  "animation-duration": "0s",
  "animation-fill-mode": "none",
  "animation-iteration-count": "1",
  "animation-name": "none",
  "animation-play-state": "running",
  "animation-timing-function": "ease",
  "background": "none",
  "background-attachment": "scroll",
  "background-blend-mode": "normal",
  "background-clip": "border-box",
  "background-color": "transparent",
  "background-image": "none",
  "background-origin": "padding-box",
  "background-position": "0% 0%",
  "background-position-x": "0%",
  "background-position-y": "0%",
  "background-repeat": "repeat",
  "background-repeat-x": "repeat",
  "background-repeat-y": "repeat",
  "background-size": "auto auto",
  "baseline-shift": "0",
  "block-size": "auto",
  "border": "medium none currentcolor",
  "border--image--slice": "100%",
  "border--image--source": "none",
  "border-block": "medium none currentcolor",
  "border-block-color": "currentcolor",
  "border-block-end": "medium none currentcolor",
  "border-block-end-color": "currentcolor",
  "border-block-end-style": "none",
  "border-block-end-width": "medium",
  "border-block-start": "medium none currentcolor",
  "border-block-start-color": "currentcolor",
  "border-block-start-style": "none",
  "border-block-start-width": "medium",
  "border-block-style": "none",
  "border-block-width": "medium",
  "border-bottom": "medium none currentcolor",
  "border-bottom-color": "currentcolor",
  "border-bottom-left-radius": "0",
  "border-bottom-right-radius": "0",
  "border-bottom-style": "none",
  "border-bottom-width": "medium",
  "border-collapse": "separate",
  "border-color": "currentcolor",
  "border-image": "none",
  "border-image-outset": "0",
  "border-image-repeat": "stretch",
  "border-image-width": "1",
  "border-inline": "medium none currentcolor",
  "border-inline-color": "currentcolor",
  "border-inline-end": "medium none currentcolor",
  "border-inline-end-color": "currentcolor",
  "border-inline-end-style": "none",
  "border-inline-end-width": "medium",
  "border-inline-start": "medium none currentcolor",
  "border-inline-start-color": "currentcolor",
  "border-inline-start-style": "none",
  "border-inline-start-width": "medium",
  "border-inline-style": "none",
  "border-inline-width": "medium",
  "border-left": "medium none currentcolor",
  "border-left-color": "currentcolor",
  "border-left-style": "none",
  "border-left-width": "medium",
  "border-radius": "0",
  "border-right": "medium none currentcolor",
  "border-right-color": "currentcolor",
  "border-right-style": "none",
  "border-right-width": "medium",
  "border-spacing": "0",
  "border-style": "none",
  "border-top": "medium none currentcolor",
  "border-top-color": "currentcolor",
  "border-top-left-radius": "0",
  "border-top-right-radius": "0",
  "border-top-style": "none",
  "border-top-width": "medium",
  "border-width": "medium",
  "bottom": "auto",
  "box-shadow": "none",
  "box-sizing": "content-box",
  "break-after": "auto",
  "break-before": "auto",
  "break-inside": "auto",
  "caption-side": "top",
  "caret-color": "auto",
  "clear": "none",
  "clip": "auto",
  "clip-path": "none",
  "clip-rule": "none",
  "color": "inherit",
  "column-count": "auto",
  "column-fill": "balance",
  "column-gap": "normal",
  "column-rule": "medium none currentcolor",
  "column-rule-color": "currentcolor",
  "column-rule-style": "none",
  "column-rule-width": "medium",
  "column-span": "auto",
  "column-width": "auto",
  "columns": "auto",
  "content": "normal",
  "counter-increment": "none",
  "counter-reset": "none",
  "css-float": "none",
  "cursor": "auto",
  "direction": "ltr",
  "display": "inline",
  "empty-cells": "show",
  "filter": "none",
  "flex": "0 1 auto",
  "flex-basis": "auto",
  "flex-direction": "row",
  "flex-flow": "row nowrap",
  "flex-grow": "0",
  "flex-shrink": "1",
  "flex-wrap": "nowrap",
  "float": "none",
  "font": "initial",
  "font-family": "inherit",
  "font-feature-settings": "normal",
  "font-optical-sizing": "auto",
  "font-size": "medium",
  "font-stretch": "normal",
  "font-style": "normal",
  "font-synthesis": "weight style",
  "font-variant": "normal",
  "font-variant-alternates": "normal",
  "font-variant-caps": "normal",
  "font-variant-east-asian": "normal",
  "font-variant-ligatures": "normal",
  "font-variant-numeric": "normal",
  "font-variant-position": "normal",
  "font-variation-settings": "normal",
  "font-weight": "normal",
  "gap": "normal",
  "grid": "none",
  "grid-area": "auto",
  "grid-auto-columns": "auto",
  "grid-auto-flow": "row",
  "grid-auto-rows": "auto",
  "grid-column": "auto",
  "grid-column-end": "auto",
  "grid-column-start": "auto",
  "grid-row": "auto",
  "grid-row-end": "auto",
  "grid-row-start": "auto",
  "grid-template": "none",
  "grid-template-areas": "none",
  "grid-template-columns": "none",
  "grid-template-rows": "none",
  "hanging-punctuation": "none",
  "height": "auto",
  "image-rendering": "auto",
  "inline-size": "auto",
  "inset": "auto",
  "inset-block": "auto",
  "inset-block-end": "auto",
  "inset-block-start": "auto",
  "inset-inline": "auto",
  "inset-inline-end": "auto",
  "inset-inline-start": "auto",
  "isolation": "auto",
  "justify-content": "normal",
  "justify-items": "legacy",
  "justify-self": "auto",
  "left": "auto",
  "letter-spacing": "normal",
  "line-break": "auto",
  "line-height": "normal",
  "list-style": "disc outside none",
  "list-style-image": "none",
  "list-style-position": "outside",
  "list-style-type": "disc",
  "margin": "0",
  "margin-block": "0",
  "margin-block-end": "0",
  "margin-block-start": "0",
  "margin-bottom": "0",
  "margin-inline": "0",
  "margin-inline-end": "0",
  "margin-inline-start": "%",
  "margin-left": "0",
  "margin-right": "0",
  "margin-top": "0",
  "mask": "none",
  "max-block-size": "0",
  "max-height": "none",
  "max-inline-size": "0",
  "max-width": "none",
  "min-block-size": "0",
  "min-height": "none",
  "min-inline-size": "0",
  "min-width": "none",
  "mix-blend-mode": "normal",
  "object-fit": "fill",
  "object-position": "50% 50%",
  "opacity": "1.0",
  "order": "0",
  "orphans": "2",
  "outline": "none",
  "outline-color": "current-color",
  "outline-offset": "none",
  "outline-style": "none",
  "outline-width": "medium",
  "overflow": "visible",
  "overflow-wrap": "normal",
  "overflow-x": "visible",
  "overflow-y": "visible",
  "padding": "0",
  "padding-block": "0",
  "padding-block-end": "0",
  "padding-block-start": "0",
  "padding-bottom": "0",
  "padding-inline": "0",
  "padding-inline-end": "0",
  "padding-inline-start": "0",
  "padding-left": "0",
  "padding-right": "0",
  "padding-top": "0",
  "page-break-after": "auto",
  "page-break-before": "auto",
  "page-break-inside": "auto",
  "paint-order": "normal",
  "perspective": "none",
  "perspective-origin": "50% 50%",
  "perspective-origin-x": "50%",
  "perspective-origin-y": "50%",
  "place-content": "normal",
  "place-items": "normal legacy",
  "place-self": "auto",
  "pointer-events": "auto",
  "position": "static",
  "quotes": "none",
  "resize": "noe",
  "right": "auto",
  "rotate": "none",
  "row-gap": "normal",
  "scale": "none",
  "scroll-margin": "0",
  "scroll-margin-bottom": "0",
  "scroll-margin-left": "0",
  "scroll-margin-right": "0",
  "scroll-margin-top": "0",
  "scroll-padding": "0",
  "scroll-padding-bottom": "0",
  "scroll-padding-left": "0",
  "scroll-padding-right": "0",
  "scroll-padding-top": "0",
  "scroll-snap-align": "none",
  "scroll-snap-type": "none",
  "shape-image-threshold": "0",
  "shape-margin": "0",
  "shape-outside": "none",
  "size": "auto",
  "src": "",
  "stroke": "none",
  "stroke-color": "current-color",
  "stroke-dasharray": "none",
  "stroke-dashoffset": "0",
  "stroke-linecap": "butt",
  "stroke-linejoin": "miter",
  "stroke-miterlimit": "4",
  "stroke-opacity": "1",
  "stroke-width": "medium",
  "tab-size": "8",
  "table-layout": "auto",
  "text-align": "start",
  "text-anchor": "start",
  "text-decoration": "none",
  "text-decoration-color": "currentcolor",
  "text-decoration-line": "none",
  "text-decoration-skip": "objects",
  "text-decoration-style": "solid",
  "text-decoration-thickness": "auto",
  "text-indent": "0",
  "text-orientation": "mixed",
  "text-overflow": "clip",
  "text-rendering": "auto",
  "text-shadow": "none",
  "text-transform": "none",
  "text-underline-offset": "auto",
  "text-underline-position": "auto",
  "top": "auto",
  "touch-action": "auto",
  "transform": "none",
  "transform-box": "view-box",
  "transform-origin": "50% 50% 0",
  "transform-origin-x": "50%",
  "transform-origin-y": "50%",
  "transform-origin-z": "0",
  "transform-style": "flat",
  "transition": "0s all ease",
  "transition-delay": "0s",
  "transition-duration": "0s",
  "transition-property": "all",
  "transition-timing-function": "ease",
  "translate": "none",
  "vector-effect": "none",
  "vertical-align": "baseline",
  "visibility": "visible",
  "white-space": "normal",
  "widows": "2",
  "width": "auto",
  "will-change": "auto",
  "word-break": "normal",
  "word-spacing": "normal",
  "word-wrap": "normal",
  "writing-mode": "horizontal-tb",
  "x": "auto",
  "y": "auto",
  "z-index": "auto",
  "zoom": "normal",
}
`;
